extension/src/shared/context/WidgetContext.tsx

```diff
@@ -81,10 +82,12 @@ export const WidgetContext: FC<PropsWithChildren<Props>> = ({
   }
 
   return (
-    <ApolloProvider client={catalogClient}>
-      <ApolloProvider client={geoClient}>
-        <ThemeProvider theme={lightTheme}>{children}</ThemeProvider>
+    <SnackbarProvider maxSnack={1}>
+      <ApolloProvider client={catalogClient}>
+        <ApolloProvider client={geoClient}>
+          <ThemeProvider theme={lightTheme}>{children}</ThemeProvider>
+        </ApolloProvider>
```

Comment:
The theme seems not be used. Maybe you need to use `SnackbarProvider` inside of `ThemeProvider`.
|This PR|Prototype|
|:--:|:--:|
|![Screenshot 2023-12-19 at 18 53 28](https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/assets/34934510/ae08082b-c3c8-43e2-8614-662d6b2399b6)|![Screenshot 2023-12-19 at 19 00 37](https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/assets/34934510/8ba6a8b3-84be-4d5d-89c0-ceaddcec2840)|